### PR TITLE
Accessibility

### LINKED ISSUE #160: Helsinki marketing needed to be added to image credit

### LINKED ISSUE #173: Infomodal: show the link URL, not just 'link' -button

### LINKED ISSUE #174: Remove hover effect from elements that cannot be clicked (mainly weather page disco effect)

### LINKED ISSUE #178: add close button on the modal

### LINKED ISSUE #179: improve alt on images and buttons with images

### PR DESCRIPTION
**Pull request by nsonb**
Updated for various minor issues

### CODE REVIEW

File: src/component/DetailedInfoModal.js [hunk 1]
```diff
@@ -241,8 +248,10 @@ const DetailedInfoModal = () => {
 
     return (
         <div style={modal}>
-            <div style={blurBackground}  onClick={toggleModal}/>
-            <div style={content} className='main-background-color'>
+            <div style={blurBackground} onClick={toggleModal}/>
+            
+            <div style={content} className='main-background-color' onClick={toggleModal}>
```
Comment (Buetix) [inline src/component/DetailedInfoModal.js]: Should the onClick be here?

**Review by Buetix — COMMENTED** [review]
The modal onClick?

File: src/component/DetailedInfoModal.js (hunk 1 above)
Comment (Buetix) [reply src/component/DetailedInfoModal.js]: Since this means clicking the modal itself closes it

Comment (nsonb) [reply src/component/DetailedInfoModal.js]: fixed the bug. It's a small mistake

**Review by Buetix — APPROVED** [review]

**Pull request merged**